### DISCUSSION TITLE
抱着进大厂的态度参与开源，有什么问题？ - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#83] [Announcements]
# 抱着进大厂的态度参与开源，有什么问题？ - 「开源之道」

https://opensourceway.community/posts/open-source-conference/2022-oscar-culture-branch-veuen-preview/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

真实录, 当然, 一小时的吐糟, 
原话肯定比这些片断多...

但是, 作为长期开源文化受益者, 尽力去宣传自己认为好的,
这本身就是开放文化行为,
值得坚持, 并不用担心当下是否有收益, 很明显这种收益是通过其它人思想转变后, 长期积累才能体现出来的,
也就是俺一直以来演讲的标准分投标题:

```
但行, 好事
莫问, 前程
```

**Reply by lijiangsheng1** [reply]
我仍然认为每一段都是切中时弊～
